sonar-python-plugin/pom.xml

```diff
@@ -146,8 +150,8 @@
             <configuration>
               <rules>
                 <requireFilesSize>
-                  <maxsize>4810000</maxsize>
-                  <minsize>4310000</minsize>
+                  <maxsize>4910000</maxsize>
```

Comment:
when I build locally I have `4917432` . Can you please increase to "4500000 - 5000000"?